Sources/XcodeProj/Scheme/XCScheme+TestableReference.swift

```diff
@@ -16,6 +22,25 @@ public extension XCScheme {
 
         // MARK: - Init
 
+        public init(skipped: Bool,
+                    parallelization: TestParallelization = .none,
+                    randomExecutionOrdering: Bool = false,
+                    buildableReference: BuildableReference,
+                    locationScenarioReference: LocationScenarioReference? = nil,
+                    skippedTests: [TestItem] = [],
+                    selectedTests: [TestItem] = [],
+                    useTestSelectionWhitelist: Bool? = nil) {
+            self.skipped = skipped
+            self.parallelization = parallelization
+            self.randomExecutionOrdering = randomExecutionOrdering
+            self.buildableReference = buildableReference
+            self.locationScenarioReference = locationScenarioReference
+            self.useTestSelectionWhitelist = useTestSelectionWhitelist
+            self.selectedTests = selectedTests
+            self.skippedTests = skippedTests
+        }
+
+        @available(*, deprecated, message: "Use init with Parallelization argument instead")
         public init(skipped: Bool,
                     parallelizable: Bool = false,
```

Comment:
Let's remove the default value, so the deprecation warning is triggered only when `parallelizable` is specified.